projects/aca-testing-shared/src/utilities/repo-client/apis/repo-api.ts

```diff
@@ -30,7 +30,7 @@ import { Logger } from '@alfresco/adf-testing';
 export abstract class RepoApi {
   alfrescoJsApi = new AlfrescoApi();
 
-  constructor(private username: string = browser.params.ADMIN_USERNAME, private password: string = browser.params.ADMIN_PASSWORD) {
+  protected constructor(private username: string = browser.params.ADMIN_USERNAME, private password: string = browser.params.ADMIN_PASSWORD) {
```

Comment:
Same question here.

Reply:
it is a good practice making abstract constructors protected, following the tooling suggestions